build_projects/shared-build-targets-utils/DependencyVersions.cs

```diff
@@ -8,5 +8,11 @@ namespace Microsoft.DotNet.Cli.Build
     public class DependencyVersions
     {
         public static readonly string CoreCLRVersion = "1.0.2-rc3-24126-01";
+
+        public static readonly string SharedFrameworkVersion = "1.0.0-rc3-004312";
```

Comment:
These versions are always going to be updated, and get out of date.  I don't think these should be here.